pyproject.toml

```diff
@@ -27,86 +27,71 @@ packages = [
 ]
 
 [tool.poetry.dependencies]
-python = ">=3.8,<3.13"
-click = ">=7.2,<=9.0"
-click-default-group = "^1.2.2"
-numpy = [
-    { version = "^1.22", python = "<3.12" },
-    { version = "^1.26", python = ">=3.12" }
-]
-Pillow = ">=9.1,<10.0"
-scipy = [
-    { version = "^1.7.3", python = "<3.12" },
-    { version = "^1.11", python = ">=3.12" }
-]
-tqdm = "^4.62.3"
-pydub = "^0.25.1"
+python = ">=3.9,<3.13"
+click = ">=8.0"
+cloup = ">=2.0.0"
+dearpygui = { version = ">=1.0.0", optional = true }
+decorator = ">=4.3.2"
+isosurfaces = ">=0.1.0"
+jupyterlab = { version = ">=3.0.0", optional = true }
+manimpango = "<=1.0.0"  # Complete API change in 1.0.0
```

Comment:
I would suggest adding a lower bound here, just like how it previously was. Also, I think shouldn't include `=` in the version, it should be `<1.0.0`.